Source/Core/DolphinWX/Main.cpp

```diff
@@ -96,21 +96,72 @@ bool DolphinApp::Initialize(int& c, wxChar **v)
 
 bool DolphinApp::OnInit()
 {
+	if (!wxApp::OnInit() || DolphinEmulatorDotComTextFileExists())
```

Comment:
> Can we rename DolphinEmulatorDotComTextFileExists to something more generic, in case we want to add stuff to it in the future?

No. You shouldn't be 'stuffing' code into functions that clearly have no logical relation with one another. If you test against a different find of file, it should have its own function.